maintainers/release-process.md

```diff
@@ -194,8 +194,54 @@ release:
 
 * Bump the version number of the release branch as above (e.g. to
   `2.12.2`).
-  
+
 ## Recovering from mistakes
 
 `upload-release.pl` should be idempotent. For instance a wrong `IS_LATEST` value can be fixed that way, by running the script on the actual latest release.
 
+## Security releases
+
+Once a security fix is ready for merging:
+
+1. Request a CVE in the [GitHub security advisory](https://github.com/NixOS/nix/security/advisories) for the security fix.
+
+1. Notify all collaborators on the advisory with a timeline for the release.
```

Comment:
In a very recent advisory, we seem to have had an issue with the reporter's access to the advisory.
This needs to be double-checked before trying to notify "all" collaborators.